LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012 Miki Tebeka <miki.tebeka@gmail.com>.
+Copyright (c) 2012 Miki Tebeka <miki.tebeka@gmail.com>, Google Inc.
```

Comment:
Why? First time someone asks me to assign copyright to them when they create a pull request.